nimha.nim

```diff
@@ -124,6 +124,7 @@ proc createDbTables() =
   filestreamDatabase(dbFile)
   mqttDatabase(dbMqtt)
   rpiDatabase(dbRpi)
+  certDatabase(dbWeb)
```

Comment:
Good catch! We could delete the `certDatabase(dbWeb)` from `web_utils.nim` then.

https://github.com/ThomasTJdev/nim_homeassistant/blob/e23df125bb2dd8b7b76b6c3b904c7e040f7442c4/nimhapkg/modules/web/web_utils.nim#L25